fpx-workers/src/data.rs

```diff
@@ -80,12 +83,12 @@ impl Store for D1Store {
     async fn span_get(
         &self,
         _tx: &Transaction,
-        trace_id: String,
-        span_id: String,
+        trace_id: &str,
+        span_id: &str,
     ) -> Result<models::Span> {
         SendFuture::new(async {
             self.fetch_one(
-                "SELECT * FROM spans WHERE trace_id=$1 AND span_id=$2",
+                self.sql_builder.span_get(),
                 &[trace_id.into(), span_id.into()],
```

Comment:
Should we maybe pass the variables into the `span_get()` function? Might make it a bit less easy for the params in the string and the args to get out of sync. 

What do you think?

Reply:
I don't see anyway to easily do that. Since we do not want to just set the variables in the sql queries (and thus expose ourselves to all kinds of SQL injection issues) and abstracting "adding of the variables" away for our current implementations (libsql and d1) already seems complicated.

Reply:
Fair points!